tests/test_data/kfoods.com/kfoods.json

```
@@ -0,0 +1,31 @@
+{
+  "author": "~Sani~",
+  "canonical_url": "https://kfoods.com/recipes/gol-gappay_urid34",
+  "site_name": "KFoods",
+  "host": "kfoods.com",
+  "title": "Gol Gappay",
+  "ingredients": [
+    "میدہ۔۔۔۔۔1کپ",
+    "تیل۔۔۔۔۔۔۔۔۔۔۔۔۔۔۔۔ 1 کھانے کا چمچ",
+    "تیل۔۔۔۔۔۔۔۔۔۔۔ فار فرائنگ",
+    "فلنگ کے لیے",
+    "آلو۔۔۔۔۔۔۔۔۔۔۔۔۔۔۔ 1کپ ( بوائلڈ)",
+    "کابلی چنے۔۔۔۔۔۔۔1کپ ( بوائلڈ) ( سفید والے)",
+    "املی۔۔۔۔۔۔۔۔۔۔ 50 گرام",
+    "چینی۔۔۔۔۔۔۔۔۔۔ 1کپ",
+    "لیموں۔۔۔۔۔۔۔۔۔۔۔۔۔ 2 عدد",
+    "سرخ مرچ پاؤڈر۔۔۔۔۔۔۔۔۔۔۔۔۔آدھا چائے چمچ",
+    "نمک۔۔۔۔۔۔۔۔۔۔۔۔آدھا چائے چمچ",
+    "پودینہ پیسٹ۔۔۔۔۔۔۔۔۔۔۔2 کھانے کے چمچ",
+    "زیرہ۔۔۔۔۔۔۔۔۔۔۔۔۔1 چائے چمچ ( توے پر بھون لیں)"
+  ],
+  "instructions_list": [
+    "میدے میں تیل ملا کر پانی کے ساتھ سخت سے گوندھ لیں۔ اور پھر اسکی پتلی سے روٹی بیل لیں۔ اس کو کٹر سے گول گول کاٹ لیں۔۔خیال رہے کہ آپ ٹکڑے ساتھ ساتھ گیلے ٹھنڈے کپڑے سے ڈانپتے جائیں۔۔۔پھر آئل گرم کر کے فرائی کر لیں۔۔۔اگر کوئی گول گپہ نہ پھولے تو اسے پاپڑی کی طرح استعمال کرلیں۔",
+    "املی کو 3-2 گھنٹے کے لئے پانی میں بھگو دیں۔۔اس کے بعد گودا نکال کر اس میں باقی چیزیں مکس کر لیں۔۔۔اور ہلکا سا جوش دے لیں۔پھر تھنڈا کرکے5-4 گلاس پانی ڈال کر کھٹاس بنا لیں۔",
+    "گول گپےکو پیش کرنے سے پہلے اس کو سائیڈ سےتھوڑا سا توڑ کے اس میں آلو اور چنے بھر لیں اور اس کھٹاس کے ساتھ پیش کریں۔"
+  ],
+  "yields": "10 servings",
+  "description": "Gol gappay recipe is served with its special chutney, boiled chickpeas, boiled potatoes and enjoyed. Access the most desirable recipes online posted on this page. You can prepare Gol Gappayonline along with the required ingredients and method so you can note them and prepare it with ease. Gol gappay is prepared and loved by every individual in Pakistan especially they are very popular among women. They are made with semolina dough and then deep fried until crispy. They are filled with chickpeas and sour water which is the specialty of gol gappay. Some people like it with sweet chutney also.",
+  "total_time": 45,
```

```suggestion
  "total_time": 45,
  "cook_time": 30,
  "prep_time": 15,
  "ratings": 5.0,
  "ratings_count": 62,
  "nutrients": {
    "servingSize": "1 Bowl",
    "calories": "100",
    "fatContent": "12"
  },
```

looks like the site provides a few more fields if you wanted to include them in the test data!